sdk/ai/azure-ai-inference/azure/ai/inference/models/_models.py

```diff
@@ -397,7 +431,7 @@ class ChatCompletionsOptions(_model_base.Model):
      yourself via a system or user message. Without this, the model may generate an unending stream
      of whitespace until the generation reaches the token limit, resulting in a long-running and
      seemingly \"stuck\" request. Also note that the message content may be partially cut off if
-     ``finish_reason=\"length\"``\\ , which indicates the generation exceeded ``max_tokens`` or the
+     ``finish_reason=\"length\"``\ , which indicates the generation exceeded ``max_tokens`` or the
      conversation exceeded the max context length."""
```

Comment:
I forget which one of the above two options (\ or \\) causes console spew warnings in Python 3.12 when you run the samples. I had to manually fix it. I reported it to Azure SDK team. No sure if what I see above is the required fix, or a revert of a manual fix that was done before and to be re-applied.